sklearn/preprocessing/_encoders.py

```diff
@@ -367,7 +367,8 @@ def _handle_deprecations(self, X):
             msg = (
                 "Passing 'n_values' is deprecated in version 0.20 and will be "
                 "removed in 0.22. You can use the 'categories' keyword "
-                "instead. 'n_values=n' corresponds to 'categories=[range(n)]'."
+                "instead. 'n_values=n' corresponds to "
+                "'categories = [range(n)] * n_features'."
```

Comment:
There should be no spaces before or after the = in a function definition (see https://lintlyci.github.io/Flake8Rules/rules/E251.html).